src/test/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/InputAvoidEscapedUnicodeCharacters5.java

```diff
@@ -0,0 +1,47 @@
+/*
+AvoidEscapedUnicodeCharacters
+allowEscapesForControlCharacters = (default)false
+allowByTailComment = true
+allowIfAllCharactersEscaped = (default)false
+allowNonPrintableEscapes = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+public class InputAvoidEscapedUnicodeCharacters5 {
+    private String a = "🎄"; // violation below
+    private String b = "\uD83E\uDD73abs😀";
+    private String c = "\uD83C\uDF84😀\uD83C\uDF84"; // comment
```

Comment:
Please make it obvious that comment allows escapes, please change comment to something like '// ok, allowed by trailing comment`

Reply:
done

Reply:
Resolved.